src/utils/ainize.ts

```diff
@@ -24,7 +25,8 @@ export const request = async <T>(
   { serviceName, opType, data, timeout = DEFAULT_TIMEOUT_MS }: AinizeRequest
 ): Promise<AinizeResponse<T>> => {
   if (!Handler.getInstance().isConnected()) {
-    throw new Error('Network error: blockchain event channel is not connected.');
+    // NOTE(jiyoung): client error handling method need to be updated.
+    throw new AinftError('unavailable', 'connection to the blockchain network could not be established.');
```

Comment:
@minjaeli the frontend is using old error message('Network error: blockchain event channel is not connected.') to check if the connection has been lost.